web/package.json

```diff
@@ -10,6 +10,7 @@
     "webpack-ts": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts",
     "webpack-dev-server-ts": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress",
     "start": "yarn run webpack-dev-server-ts --env.config_name=integreat-test-cms --env.dev_server",
+    "start:https": "yarn run webpack-dev-server-ts --https --env.config_name=integreat-test-cms --env.dev_server",
```

Comment:
I think it would be easier to run "start:malte --https".

To make it easy to remember this we could add the following text to the error message if enabling geolocation fails:

"To use geolocation in a development build you have to start the dev server with "--https" like "yarn start --https"

Reply:
I agree. I just think we would have to add additional code which we don't need. I don't see any function in the API doc which handles this. We could add an useEffect and try to get the navigator.location and if that fails we throw an error, but i don't know if we should add this just to get error message

Reply:
i added some console.log and removed script